site/documentation/content/concepts/device-identity.md

```diff
@@ -12,7 +12,7 @@ Hono does not make any assumptions about the format of a device identifier (or *
 
 ## Tenant
 
-Hono supports the logical partitioning of devices into groups called *tenants*. Each tenant has a unique identifier, a string called the *tenant-id*, and can be used to provide a logical grouping of devices belonging e.g. to the same application scope or organizational unit. Each device can thus be uniquely identified by the tuple (*tenant-id*, *device-id*). This tuple is broadly used throughout Hono's APIs when addressing a particular device.
+Hono supports the logical partitioning of devices into groups called [*tenants*]({{< relref "/concepts/tenancy" >}}). Each tenant has a unique identifier, a string called the *tenant-id*, and can be used to provide a logical grouping of devices belonging e.g. to the same application scope or organizational unit. Each device can thus be uniquely identified by the tuple (*tenant-id*, *device-id*). This tuple is broadly used throughout Hono's APIs when addressing a particular device.
 
 ## Device Registration
```

Comment:
This section sounds as if Hono always requires authentication of devices. In fact, this can currently be turned off for most protocol adapters.